src/components/ProfileView.jsx

```diff
@@ -154,7 +154,7 @@ const ProfileView = props => {
             </div>
           </Stack>
           <div className="u-mt-2">
-            <DeleteAccount />
+            <DeleteAccount email={instance?.data?.attributes?.email} />
```

Comment:
You should not check if instance.data.attributes are there. You're in a `!isFetching` condition. This `isFetching` is set when the `fetchInfos` is launched. After that, is set to false when the request is done. 

Also it's weird to be able to pass `undefined` to DeleteAccount, since we'll display the modal without any email.